### PR TITLE
update: Change the output method of index space analysis and return ObdiagResult model

### PR DESCRIPTION
**Pull request by wayyoungboy**
update: Change the output method of index space analysis and return ObdiagResult model

### CODE REVIEW

File: diag_cmd.py [hunk 1]
```diff
@@ -265,12 +265,17 @@ def do_command(self):
...
+                ret = self._do_command(obdiag)
+                exit_code = 0
+            except Exception as e:
+                ret = ObdiagResult(code=ObdiagResult.SERVER_ERROR_CODE, error_data="command failed. Please contact oceanbase community. e: {0}".format(e))
+                exit_code = 1
             # if silent is true ,print ret
             if ROOT_IO.silent:
-                if isinstance(ret, ObdiagResult) is False:
-                    ROOT_IO.error('The return value of the command is not ObdiagResult. Please contact thebase community. The return value is: {0}'.format(ret))
+                if ret is None or isinstance(ret, ObdiagResult) is False:
```
Comment (frf12) [inline diag_cmd.py:277]: `isinstance(None, ObdiagResult) is False` 这个条件也是成立的，为什么要单独判断下ret is None

File: diag_cmd.py [hunk 2]
```diff
@@ -265,12 +265,17 @@ def do_command(self):
...
+                    ROOT_IO.error('The return value of the command is not ObdiagResult. Please contact oceanbase community. The return value is: {0}'.format(ret))
```
Comment (frf12) [inline diag_cmd.py:278]: oceanbase 应写作 OceanBase，组织名称要按照规范来

File: diag_cmd.py (hunk 1 above)
Comment (wayyoungboy) [reply diag_cmd.py]: fixed

File: diag_cmd.py (hunk 2 above)
Comment (wayyoungboy) [reply diag_cmd.py]: fixed

**Review by frf12 — APPROVED** [review]
LGTM

**Pull request merged**